README.md

```diff
@@ -1,22 +1,34 @@
-# Cellar Rebalancer
+# Steward
 
-The Cellar Rebalancer is an appliaction intended for developers and validators on Sommelier network.
+Steward is an application intended for developers and validators on Sommelier network.
 
 It can run in single signer mode or as a voter in the Cosmos sommelier protocol.
```

Comment:
I would suggest reversing this, perhaps along the lines of "It can run as a voter in the Cosmos Sommelier protocol or in test mode to directly interact with Ethereum contracts as a single signer."

Reply:
I don't understand what you mean with "reverse".

Reply:
Ah, I meant in this sentence starting with its primary purpose rather than how it can be used in test.

Reply:
done!